scripts/remove-kubestellar

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
This should aim to do the opposite of the bootstrap script (https://github.com/kcp-dev/edge-mc/blob/main/hack/bootstrap-kubestellar.sh).  Undoing the other stuff belongs elsewhere.

We can defer removing the excesses.  But right now there are also shortcomings.  The current revision does not remove the downloaded directories.